src/analysis/convex/basic.lean

```diff
@@ -604,6 +670,46 @@ begin
     exact (@h (x, f x) (y, f y) ⟨hx, le_refl _⟩ ⟨hy, le_refl _⟩ a b ha hb hab).2 }
 end
 
+/-- If a function is convex on s, it remains convex when prepended by an affine map -/
+lemma convex_on.affine_preimage {f : F → ℝ} {g : affine_map ℝ E E F F} {s : set F}
```

Comment:
I'd call it `convex_on.comp_affine_map`. I think that `g` should be an explicit argument.

Reply:
Done!